docs/customising/config.md

```diff
@@ -206,11 +206,32 @@ indentation correct. If in doubt, look at the examples already in the file, and
     <a name="force_registration_on_new_request"><code>FORCE_REGISTRATION_ON_NEW_REQUEST</code></a>
   </dt>
   <dd>
-    Does a user needs to sign in to start the New Request process?
+    Does a user need to sign in to start the New Request process?
```

Comment:
Maybe we could highlight "start" here:

``
Does a user need to sign in to _start_ the New Request process?
```